service/tablet_allocator.cc

```diff
@@ -1023,7 +1023,6 @@ class load_balancer {
 
             target = nodes_by_load_dst.back();
             auto& target_info = nodes[target];
-            auto& src_info = nodes[src.host];
```

Comment:
nit: the `src` itself becomes almost unused after that

Reply:
It's still used though, what do you suggest to do?

Reply:
It's used in two log messages and in preparation of `auto mig = tablet_migration_info {kind, source_tablet, src, dst};`. I'd just prepare the `{src_host, src_shard}` in place to keep less local variables